<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDaily">
    <property name="title">
     <string>Daily</string>
    </property>
    <addaction name="actionChange_daily_tasks"/>
    <addaction name="actionAdd_notes"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionShow_history"/>
    <addaction name="actionLevels"/>
    <addaction name="actionBudget"/>
    <addaction name="actionNotes_2"/>
   </widget>
   <widget class="QMenu" name="menuToDo_tasks">
    <property name="title">
     <string>ToDo tasks</string>
    </property>
    <addaction name="actionadd_a_task"/>
    <addaction name="actionMark_task_as_finished"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="actionBackUp"/>
   </widget>
   <addaction name="menuDaily"/>
   <addaction name="menuHistory"/>
   <addaction name="menuToDo_tasks"/>
   <addaction name="menuCreate"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChange_daily_tasks">
   <property name="text">
    <string>Manage daily tasks</string>
   </property>
  </action>
  <action name="actionShow_history">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionadd_a_task">
   <property name="text">
    <string>Add a task</string>
   </property>
  </action>
  <action name="actionMark_task_as_finished">
   <property name="text">
    <string>Mark a task as finished</string>
   </property>
  </action>
  <action name="actionLevels">
   <property name="text">
    <string>Levels</string>
   </property>
  </action>
  <action name="actionBudget">
   <property name="text">
    <string>Budget</string>
   </property>
  </action>
  <action name="actionBackUp">
   <property name="text">
    <string>Database BackUp</string>
   </property>
  </action>
  <action name="actionNotes">
   <property name="text">
    <string>Daily Notes</string>
   </property>
  </action>
  <action name="actionAdd_notes">
   <property name="text">
    <string>Add notes</string>
   </property>
  </action>
  <action name="actionNotes_2">
   <property name="text">
    <string>Notes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
